napari/utils/_testsupport.py

```diff
@@ -203,6 +205,9 @@ def make_napari_viewer(
     settings = get_settings()
     settings.reset()
 
+    _initialize_plugins.cache_clear()
+    init_qactions.cache_clear()
```

Comment:
the `lru_cache` persists beyond this fixture. We could also do something like `@lru_cache(maxsize=0 if "pytest" in sys.modules else 256)` so the cache is essentially deactivated when pytest runs.

Reply:
This could be conditionally run via param, to save time, though few plugins are used in tests.